EXILED/Exiled.API/Features/CustomStats/CustomHumeShieldStat.cs

```diff
@@ -0,0 +1,79 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="CustomHumeShieldStat.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.CustomStats
+{
+    using Mirror;
+    using PlayerRoles.PlayableScps.HumeShield;
+    using PlayerStatsSystem;
+    using UnityEngine;
+    using Utils.Networking;
+
+    /// <summary>
+    /// A custom version of <see cref="HumeShieldStat"/> which allows the player's max amount of HumeShield to be changed.
+    /// </summary>
+    public class CustomHumeShieldStat : HumeShieldStat
+    {
+        /// <inheritdoc />
+        public override float MaxValue => CustomMaxValue == -1 ? base.MaxValue : CustomMaxValue;
+
+        /// <summary>
+        /// Gets or sets the multiplier for gaining HumeShield.
+        /// </summary>
+        public float ShieldRegenerationMultiplier { get; set; } = 1;
+
+        /// <summary>
+        /// Gets or sets the maximum amount of HumeShield the player can have.
+        /// </summary>
+        public float CustomMaxValue { get; set; } = -1;
+
+        private float ShieldRegeneration => TryGetHsModule(out HumeShieldModuleBase controller) ? controller.HsRegeneration * ShieldRegenerationMultiplier : 0;
+
+        /// <inheritdoc/>
+        public override void Update()
+        {
+            if (MaxValue == -1)
+            {
+                base.Update();
+                return;
+            }
+
+            if (!NetworkServer.active)
+                return;
+
+            if (_valueDirty)
+            {
+                new SyncedStatMessages.StatMessage()
+                {
+                    Stat = this,
+                    SyncedValue = CurValue,
+                }.SendToHubsConditionally(CanReceive);
+                _lastSent = CurValue;
+                _valueDirty = false;
+            }
+
+            if (ShieldRegeneration == 0)
```

Comment:
Cool be interesting to override this value

Reply:
I made a multiplier for it